components/button/button-icon.js

```diff
@@ -62,6 +62,7 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 					--d2l-button-icon-h-align: calc(((2rem + 2px - 0.9rem) / 2) * -1);
 					--d2l-icon-fill-color: var(--d2l-button-icon-fill-color, var(--d2l-color-tungsten));
 					display: inline-block;
+					line-height: 0;
```

Comment:
What was happening that required this change?

Reply:
The height of the button icon was getting an extra pixel since it was using `line-height` instead of the icon height.

Reply:
I think this should be fine. Though I get 42x42px with or without this. 🤔

Reply:
Weird, it definitely was busted without it for me earlier. See thread: https://d2l.slack.com/archives/G03Q166G2/p1669911968210739

Reply:
Ok, yeah. I think the custom dimensions are an important factor there.